### PR TITLE
Githubstudent muskiter

### PR DESCRIPTION
**Pull request by BrunoSonego**
Adding Sao Leo to the map

Closes #940

### CODE REVIEW

File: _pins/byassine93.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ ---
+ githubHandle: byassine93
+ pageUrl: byassine93.github.io
+ timestamp: TODAYS-DATE-2017-06-05
+ ---
```
Comment (hectorsector) [inline _pins/byassine93.yaml]: @BrunoSonego remove this file since it belongs to another student

File: _pins/brunosonego.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+ githubHandle: brunosonego
+ latitude: -29.764830
+ longitude: -51.144323
```
Comment (hectorsector) [inline _pins/brunosonego.json]: remove the indentations not his file

File: _pins/byassine93.yaml (hunk 1 above)
Comment (BrunoSonego) [inline _pins/byassine93.yaml]: Cant remove. i'm not on the branch

File: _pins/brunosonego.json (hunk 2 above)
Comment (BrunoSonego) [inline _pins/brunosonego.json]: Didn't worked, or i've done it wrong

**Comment by hectorsector** [comment]
@BrunoSonego sorry about the headaches! I've gone ahead and removed that extra file which gave you the ✅. Feel free to merge whenever you're ready 🎉

**Pull request merged**